src/ZipkinBundle/Components/Messenger/ZipkinStamp.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+class ZipkinStamp implements StampInterface
+{
+    public $context = [];
+
+    public function getContext(): array
+    {
+        return $this->context;
+    }
+
+    public function add(string $key, ?string $value): void
```

Comment:
Whether we keep it or not this method I think `put` is more appropriate name.

Reply:
Do you think is better if I implement `RemoteSetter` interface? So I can replace `ZipkinStamp::add` with `put` method.